src/main/java/org/apache/accumulo/examples/reservations/ARS.java

```diff
@@ -301,14 +301,15 @@ public void run() {
       } else if (tokens[0].equals("quit") && tokens.length == 1) {
         break;
       } else if (tokens[0].equals("connect") && tokens.length == 6 && ars == null) {
-        try (AccumuloClient client = Accumulo.newClient().to(tokens[1], tokens[2])
-            .as(tokens[3], tokens[4]).build()) {
-          if (client.tableOperations().exists(tokens[5])) {
-            ars = new ARS(client, tokens[5]);
-            reader.println("  connected");
-          } else
-            reader.println("  No Such Table");
-        }
+        // its good to close clients when possible, but it was not done here for simplicity
```

Comment:
This comment could be improved to explain why the client needs to be left open here. I don't think it's merely a matter of "simplicity".

Reply:
> This comment could be improved to explain why the client needs to be left open here.

@ctubbsii what wording would you suggest?

Reply:
Initially, I didn't understand why the client needed to be open here, so I was suggesting an improvement in the comment that would explain that. However, @keith-turner  explained it to me out of band, so now I understand why and can suggest the following:

```suggestion
        // the client can't be closed here, because it is passed to the new ARS object and keeping it open is simpler than adding code to close it later
```

Reply:
Or, more succinctly:

```suggestion
        // the client can't be closed here, because it is passed to the new ARS object
```

Reply:
Thank you, I will change the comment